test-integration/operator-tests/src/test/resources/infinispans/client_validation.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: infinispan.org/v1
+kind: Infinispan
+metadata:
+  name: client-validation
+  annotations:
+    infinispan.org/monitoring: 'false'
+spec:
```

Comment:
Please add `service: type: DataGrid` to the `spec`. Default is `Cache` which is meant only for simplest use cases and planed to be deprecated anyway

Reply:
Added!